### DISCUSSION TITLE
Volleyball Matches directed using OBS Blade

### DISCUSSION THREAD

**Discussion by filippoadessi** [Kounex/obs_station#79] [Show and tell]
Hi, I'm using OBS Blade to actively direct our volleyball teams matches.
[Last match directed](https://www.youtube.com/live/rN9E5zAlei8?si=Lo7eZmr-kY4RIiG4)

**Comment by alanbjoseph** [comment]
the stream is private.
cannot access.

**Reply by filippoadessi** [reply]
check it out right now! I’ve **updated** youtube link.

**Comment by alanbjoseph** [comment]
> Hi, I'm using OBS Blade to actively direct our volleyball teams matches.
> [Last match directed](https://www.youtube.com/live/rN9E5zAlei8?si=Lo7eZmr-kY4RIiG4)

you aren't switching any scenes in the stream.
are you controlling the scorecard using obs blade?
if so, how are you doing it?

**Comment by filippoadessi** [comment]
here https://www.youtube.com/live/tb84p3s10Lo?si=3uqs2R1prQ4b6c_x you can appreciate scene switching.

Scorebug is updated using a dedicated application.

I’m looking for an obs blade enhancement which enable a virtual ptz using gesture of a source that is bigger than the scene canvas.
Il 1 ago 2024, 04:54 +0200, alanbjoseph ***@***.***>, ha scritto:
